static/js/emoji_picker.js

```diff
@@ -431,14 +431,15 @@ exports.navigate = function (event_name) {
     var is_filter_focused = $('.emoji-popover-filter').is(':focus');
     var next_section = 0;
     // special cases
-    if (is_filter_focused && event_name === 'down_arrow') {
-        // move down into emoji map
+    if (is_filter_focused && (event_name === 'down_arrow' || event_name === 'right_arrow')) {
+        // Move down into emoji map.
         selected_emoji.focus();
         if (current_section === 0 && current_index < 6) {
             $(".emoji-popover-emoji-map").scrollTop(0);
         }
         return true;
```

Comment:
This isn't right; we only want right-arrow to move down if we're already at the right end.  Now one can't arrow back and forth within the text box :/